packages/block-editor/src/components/block-list/block-elements.js

```diff
@@ -0,0 +1,18 @@
+const elements = [
```

Comment:
Since this is a constant, [it should use all-caps in its name](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#constants).